src/main/java/net/neoforged/neoform/runtime/actions/InjectFromZipFileSource.java

```diff
@@ -34,15 +35,25 @@ public class InjectFromZipFileSource implements InjectSource {
      */
     @Nullable
     private final Pattern includeFilterPattern;
+    /**
+     * This function can modify the content that is being copied.
+     */
+    @Nullable
```

Comment:
Instead of making this nullable, default it to `(entry, in, out) -> in.transferTo(out)`